packages/constellation/src/components/Compositions/Table/SimpleTable/SimpleTable.stories.tsx

```diff
@@ -0,0 +1,45 @@
+import { ComponentMeta, ComponentStory } from '@storybook/react'
+import React, { FC } from 'react'
+
+import { SimpleTable as SimpleTableComponent } from '.'
+
+export default {
+  argTypes: {},
+  args: {},
+  component: SimpleTableComponent,
+  title: 'Compositions/Table/SimpleTable',
+} as ComponentMeta<FC>
+
+const Template: ComponentStory<FC> = () => {
+  return (
+    <SimpleTableComponent
+      header={[
+        { cellType: 'text', text: 'Asset' },
+        { cellType: 'text', subtext: 'Amount' },
+        { cellType: 'text', subtext: 'Actions' },
+      ]}
+      rows={[
+        [
+          { cellType: 'text', subtext: 'BTC', text: 'Bitcoin' },
+          { cellType: 'text', subtext: '1 BTC', text: '$32,000' },
+          {
+            cellType: 'button',
+            primaryButton: { onClick: () => {}, text: 'Buy', variant: 'primary' },
+            secondaryButton: { onClick: () => {}, text: 'Sell', variant: 'secondary' },
+          },
+        ],
+        [
+          { cellType: 'text', subtext: 'BTC', text: 'Bitcoin' },
+          { cellType: 'text', subtext: '1 BTC', text: '$32,000' },
+          {
+            cellType: 'button',
+            primaryButton: { onClick: () => {}, text: 'Buy', variant: 'primary' },
+            secondaryButton: { onClick: () => {}, text: 'Sell', variant: 'secondary' },
+          },
+        ],
+      ]}
```

Comment:
I think the api for this component risks becoming a little unwieldy because we are essentially serializing the children row and cell components and their props into object data that is then passed to the parent `SimpleTableComponent`.

The pros of this approach are that we lock down a lot of the internals of the table (which can sometimes be a double-edged sword). The cons are that it can be a little complex populating the table with children. Also it requires the consumer knowing about these additional data structures and how to pass them in.

An alternative here if we're ok with the table being a little more open-ended would be a more composable approach, where we explicitly compose the row and cell components as children, like:
```jsx
<SimpleTableComponent header={...}>
  <Row><BaseCell /></Row>
  <Row><BaseCell /></Row>
</SimpleTableComponent>```

The benefit would be that you're using the React components you've already created as opposed building up data structures to pass as props. IMO this makes creating a table and managing its children slightly simpler. The con is that the table children are slightly less locked down in terms of what can be passed to the Table.

Reply:
In the example above I left the header as a prop because it felt like more of a one-off unique to the table component that probably won't grow in complexity the same way the rows/cells might.

Reply:
I really like this idea actually. For simple tables this makes it much more readable and for larger datasets the row component would still work very well with `react-table`.